plugins/daapclient/client.py

```diff
@@ -42,7 +42,7 @@ def _get_response(self, r, params={}, gzip=1):
 
         log.debug('getting %s', r)
 
-        headers = {'Client-DAAP-Version': '3.0', 'Client-DAAP-Access-Index': '2'}
+        headers = {'User-Agent': 'Exaile/4.x', 'Client-DAAP-Version': '3.0', 'Client-DAAP-Access-Index': '2'}
```

Comment:
Use exaile.get_user_agent_string() instead. 

https://github.com/exaile/exaile/blob/8fb0ebb4ea108792d38e2b8ad355b27d70e1d246/xl/main.py#L880

Reply:
Ah, that's indeed much better - thanks!